yarn.lock

```diff
@@ -16994,7 +16992,6 @@ pathval@^1.1.0:
     react-toastify "^5.3.1"
     react-tooltip "^3.10.0"
     sharedb "^1.1.0"
-    tippy.js "^6.3.1"
```

Comment:
does seem like it was removed here @adamabeshouse @onursumer.  @onursumer should it have been?

Reply:
@alisman @adamabeshouse This is still a dependency for pathway mapper, not sure why it was removed. There might be an issue with the latest pathway mapper version.